docs/guides/bypass-11.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: default
+title: Windows 11 Local Account Setup 
+nav_exclude: false
+parent: General Guides
+has_children: false
+search_exclude: false
+last_modified_date: 2024-06-27
+---
+# Windows 11 Local Account Setup 
+{: .no_toc }
+
+{% include toc.md %}
+
+# Introduction
+After you have installed Windows 11, you will enter the Out-Of-The-Box experience (a.k.a OOBE). From here, you can simply follow the prompts on screen and setup Windows. 
+
+However, there are certain cases where you'll need to bypass the network requirements or the account requirements (i.e. you have no internet or no Microsoft account). In these cases, you need to make a local account and follow the steps shown below. The process will differ based on whether you have the Home Edition or Pro Edition.
+
+## Bypassing network requirements 
```

Comment:
Capitalization of sections is inconsistent on this page, I believe this method of not capitalizing every word is typically what we use in the Wiki